calculator_test.go

```diff
@@ -2,23 +2,304 @@ package calculator_test
 
 import (
 	"calculator"
+	"math/rand"
 	"testing"
 )
 
 func TestAdd(t *testing.T) {
-	t.Parallel()
-	var want float64 = 4
-	got := calculator.Add(2, 2)
-	if want != got {
-		t.Errorf("want %f, got %f", want, got)
+
+	t.Run("Fixed number addition", func(t *testing.T) {
```

Comment:
We should have a few more test cases here, really, shouldn't we? Ideally, a mix of different numbers of inputs, from 2 to many.

Reply:
but we have sum four numbers and addRandom. I think it's enough

Reply:
Right, but part of the point of this exercise is to get you into the habit of writing table tests for all your functions. Let's do the same here for Add as we have for all the other functions.